test/integration/query-interface.test.js

```diff
@@ -11,8 +11,9 @@ const current = Support.sequelize;
 const _ = require('lodash');
 
 describe(Support.getTestDialectTeaser('QueryInterface'), () => {
+
   beforeEach(function () {
-    this.sequelize.options.quoteIdenifiers = true;
+    this.sequelize.options.quoteIdentifiers = true;
```

Comment:
Could you look into moving this `quoteIdentifiers` change to another PR? To lighten this PR